.github/workflows/pull-request.yaml

```diff
@@ -63,7 +88,8 @@ jobs:
           path: |
             packages/**/test/**/failures/
 
-      - run: melos run test-update-goldens
+      - name: Run golden tests
+        run: melos run test-update-goldens
```

Comment:
Not related to this PR, but do we want to run `test-update-goldens`? 🤔 

What's the benefit of updating the golden tests in CI?

Reply:
Since this repo does not use docker to run golden tests,  currently the way to update goldens is to get them from the CI artifacts. This step only runs if the `melos run test` above fails. 

It's one of the things on the list to improve in our OSS workflows.